### PR TITLE
Upgrade ckeditor to version 34.2.0 and postcss to version 8

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no 
| New feature? |  yes
| BC breaks? | no 
| Deprecations? | no  
| Fixed tickets | parts of #6343 
| Related issues/PRs | # 
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Upgrade ckeditor to version 34.2.0.

#### Why?

Supporting the latest ckeditor features and plugins.

#### TODO

 - [x] https://github.com/madeleineostoja/postcss-hexrgba/issues/31 all `rgba($` do not longer work

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/CKEditor5/CKEditor5.js [hunk 1]
```diff
@@ -197,8 +198,8 @@ export default class CKEditor5 extends React.Component<Props> {
                     },
                 } = this.editorInstance;
 
-                this.editorInstance.isReadOnly = disabled;
                 if (disabled) {
+                    this.editorInstance.enableReadOnlyMode('disabled');
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/CKEditor5/CKEditor5.js:202]: https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-editor-isreadonly-has-no-setter

**Comment by alexander-schranz** [comment]
There are 2 ways of getting this merged. First would be create a fork of the used postcss plugin based on existi compatible pull request here: https://github.com/madeleineostoja/postcss-hexrgba/pull/33
Second one would be go to alternative plugin. Like the following but it does make the code a lot harder to read and requires a rewrite of all rgba usages currently https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-color-hex-alpha.

I personally would go with fork exist postcss-hexrgba package.

/cc @sulu/core-team

**Comment by niklasnatter** [comment]
> There are 2 ways of getting this merged. First would be create a fork of the used postcss plugin based on existi compatible pull request here: [madeleineostoja/postcss-hexrgba#33](https://github.com/madeleineostoja/postcss-hexrgba/pull/33) Second one would be go to alternative plugin. Like the following but it does make the code a lot harder to read and requires a rewrite of all rgba usages currently https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-color-hex-alpha.

Looks like https://github.com/madeleineostoja/postcss-hexrgba/pull/33 was merged a few hours ago, so we dont need to fork 🥳

File: UPGRADE.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
 
 ## 2.5.0
 
+### JS Dependencies update
```
Comment (niklasnatter) [inline UPGRADE.md]:
```suggestion
### Updated JavaScript dependencies
```

File: UPGRADE.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
...
+
+Sulu requires now the following new versions JS packages for admin build:
```
Comment (niklasnatter) [inline UPGRADE.md:7]: The JavaScript dependencies of the Sulu administration interface were updated to the following versions:

File: UPGRADE.md [hunk 4]
```diff
@@ -2,6 +2,26 @@
...
+ - `postcss-hexrgba`: `^2.0.0`
+ - `postcss-import`: `^14.1.0`
+ - `postcss-loader`: `^4.0.0`
+ - `postcss-nested`: `^5.0.6`
+ - `postcss-simple-vars`: `^6.0.3`
+ - `autoprefixer`: `^10.4.7`
+
+If you created custom admin components you need to upgrade also
+your components. The core js will automatically be update
+via the [update build](https://docs.sulu.io/en/latest/upgrades/upgrade-2.x.html) command.
```
Comment (niklasnatter) [inline UPGRADE.md]:
```suggestion
If you have integrated custom JavaScript components into the administration interface,
you might need to adjust your components to be compatible with the updated dependencies.
If you have not integrated custom JavaScript code, you project is adjusted automatically by the
[update build](https://docs.sulu.io/en/latest/upgrades/upgrade-2.x.html) command.
```

File: UPGRADE.md (hunk 3 above)
Comment (niklasnatter) [inline UPGRADE.md]:
```suggestion
The JavaScript dependencies of the Sulu administration interface were updated to the following versions:
```

**Pull request merged**